1.0/warn-objects/warn03_zero_padded_versions/v0003/inventory.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "bb123cd4567",
-  "head": "v0003",
+  "head": "v0004",
```

Comment:
Same as above; this is the inventory that was current when v0003 was most recent version.

Reply:
Addressed by 1d14868